lib/yoshi.ml

```diff
@@ -34,7 +34,7 @@ module Gen = struct
 
   let rec string_of_field_value value =
     match value with
-    | `String s -> Printf.sprintf "\"%s\"" s
+    | `String s -> Printf.sprintf "{js|%s|js}" s
```

Comment:
I'm wondering if we need the custom delimiter. Does `{|...|}` work on ocaml.org?

Reply:
Wait... that's a working syntax too? 🙈🤣 Then yes, I would expect that works. All ocaml.org was choking on was a `"` character and the only syntax I knew was what I did there. 🤣

Reply:
It might be that it doesn't work, IIRC, I added the custom delimiter because some strings contained `|}` (which makes sense if they contain OCaml code). 
A better approach might be to escape the characters in the strings.

Reply:
Ah ok, yes, the markdown documents do contain OCaml code. 🤔 So it's the custom delimiter or escaping all double quotes?